src/main/java/com/sesac/carematching/community/post/Post.java

```diff
@@ -0,0 +1,68 @@
+package com.sesac.carematching.community.post;
+
+import com.sesac.carematching.community.category.Category;
+import com.sesac.carematching.user.User;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.ColumnDefault;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import java.time.Instant;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "COMMUNITY_POST")
+public class Post {
+    @Id
+    @Column(name = "CPNO", nullable = false)
+    private Integer id;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JoinColumn(name = "UNO", nullable = false)
+    private User uno;
```

Comment:
```suggestion
    private User user;
```
이 부분은 User 클래스이니 uno가 아니라 user라고 이름을 바꾸는거 어떤가요?

Reply:
이런 식으로 다 바꿨는데, chat_room 테이블에 있는 request_uno(User 엔티티)랑 receive_uno(Caregiver 엔티티)를 어떻게 쓸지 에매하네요

Reply:
현재 써있는 'requester'와 'caregiver' 좋은 것 같아요.